moninag/tests/unittests/registration/test_view.py

```diff
@@ -0,0 +1,316 @@
+import json
+
+from django.core.urlresolvers import reverse
+from django.test import TestCase, Client, mock
+
+from registration.models import CustomUser
+
+
+class Fakeuidb64(object):
+    """Mock uidb64 for password reset"""
+
+    def __init__(self, *args, **kwargs):
+        pass
+
+    def uidb64(self):
+        return '15'
```

Comment:
Why do you choose 15? :slightly_smiling_face: 
I ask, because it matters :upside_down_face: